micrometer-spring-legacy/src/test/java/io/micrometer/spring/async/ThreadPoolTaskExecutorMetricsTest.java

```diff
@@ -15,7 +15,10 @@
  */
 package io.micrometer.spring.async;
 
-import io.micrometer.core.instrument.*;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.MockClock;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
```

Comment:
I love that this test went from a wildcard and the compatibility kit went back to a wildcard import.

Are you sure you don't want to adopt a style of `no wildcard imports except for static imports`?

Reply:
Both follow the code style:

![image](https://user-images.githubusercontent.com/1697736/35106559-29da5f96-fc34-11e7-923e-bd67a626c180.png)

The compatibility kit has 5 imports in `io.micrometer.core.instrument` and this test has 4. I guess I understand the way it looks in a Git diff when you go from 5 imports to 4 and see the import exploded, but I just don't find that distracting enough to warrant a custom code style that I have to remember to apply.

It just uses IntelliJ defaults right now. As discussed earlier this week with @mweirauch, I know this is somewhat inconvenient for Eclipse users, but this is one of those "damned if you do, damned if you don't" sort of things.